org.contextmapper.dsl/src/org/contextmapper/tactic/dsl/TacticDDDLanguage.xtext

```diff
@@ -419,8 +426,15 @@ Reference :
      ("orderby" "=" orderBy=STRING)? &
      ((orderColumn?="orderColumn") ("=" orderColumnName=STRING)?)? &
      (oppositeHolder=OppositeHolder)?)
+     (ASC associationLabel=STRING)?
     (";")?;
 
+Association :
```

Comment:
@stefan-ka did you have a chance to think this over?

Reply:
I see your point @StevenVanDijk. So once I model the data structures, the semantic relationships can still be removed to avoid duplicate relations in the visualization. From that point of view I find it good.